publiccode.yaml

```diff
@@ -0,0 +1,50 @@
+publiccodeYmlVersion: "0.2"
+name: Signalen frontend
+url: "https://github.com/signalen/frontend"
+softwareVersion: "1"
+releaseDate: "2021-05-06"
+platforms:
+  - web
+categories:
+  - it-development
```

Comment:
Perhaps `workflow-management`, `task-management` and `communications` are more applicable.

Reply:
Agreed,
I tried to fill it out as correctly as i could.
But there will be thing that have to be changed for sure.